cognite/client/data_classes/_base.py

```diff
@@ -220,9 +220,9 @@ def to_pandas(
             pandas.DataFrame: The dataframe.
         """
         pd = local_import("pandas")
-        from cognite.client.data_classes.data_modeling.typed_instances import _TypedInstance
+        from cognite.client.data_classes.data_modeling.typed_instances import TypedInstance
 
-        if isinstance(self, _TypedInstance):  # TODO: This is hacky, but it works for now
+        if isinstance(self, TypedInstance):  # TODO: This is hacky, but it works for now
             dumped = self.dump(camel_case=camel_case, use_attribute_name=not camel_case)  # type: ignore [call-arg]
```

Comment:
This doesn't make any sense to me. What does `use_attribute_name` have to do with camel casing?

Reply:
In the Typed classes, we use snake_case version of the view properties. But when these are dumped, they are typically camelCase such that they can be written to the API.

When we do `.to_pandas` a Data Scientist would expect the columns to match the attributes of the Typed Instance and thus this workaround.

Reply:
"typically" camelCase

Reply:
what about cases where they're not?